pkgs/applications/networking/mailreaders/neomutt/default.nix

```
@@ -73,42 +73,47 @@ stdenv.mkDerivation rec {
   ''
   # https://github.com/neomutt/neomutt-contrib
   # Contains vim-keys, keybindings presets and more.
-  + lib.optionalString withContrib "${lib.getExe lndir} ${passthru.contrib} $out/share/doc/neomutt";
+  + lib.optionalString withContrib "${lib.getExe lndir} ${finalAttrs.passthru.contrib} $out/share/doc/neomutt";
 
   doCheck = true;
 
   preCheck = ''
-    cp -r ${passthru.test-files} $(pwd)/test-files
+    cp -r ${finalAttrs.passthru.test-files} $(pwd)/test-files
+
     chmod -R +w test-files
     (cd test-files && ./setup.sh)
 
     export NEOMUTT_TEST_DIR=$(pwd)/test-files
+
+    # The test fails with: node_padding.c:135: Check rc == 15... failed
+    substituteInPlace test/main.c \
+      --replace-fail "NEOMUTT_TEST_ITEM(test_expando_node_paddingest)" ""
```

```suggestion
      --replace-fail "NEOMUTT_TEST_ITEM(test_expando_node_padding)" ""
```

see https://github.com/neomutt/neomutt/blob/20240425/test/main.c#L313

```
Error: no ID for constraint linkend: "mix-entry-format".
LC_ALL=C w3m -T text/html -I utf-8 -O utf-8 -dump docs/manual.html > docs/manual.txt || \
LC_ALL=C lynx -localhost -dump -nolist -nonumbers -with_backspaces \
        -display_charset=us-ascii docs/manual.html > docs/manual.txt || \
LC_ALL=C elinks -dump -no-numbering -no-references \
        docs/manual.html | sed -e 's,\\001, ,g' > docs/manual.txt
Running phase: checkPhase
substituteStream(): ERROR: pattern NEOMUTT_TEST_ITEM\(test_expando_node_paddingest\) doesn't match anything in file 'test/main.c'
/nix/store/xfhkjnpqjwlf6hlk1ysmq3aaq80f3bjj-stdenv-linux/setup: line 131: pop_var_context: head of shell_variables not a function context
error: builder for '/nix/store/28li12nx981v9zx43yrwgkcm8gr768dk-neomutt-20240425.drv' failed with exit code 1;
```

---

Seems like introduced a copy & waste error… ;) Thanks for double-checking and apologies for the turn-around…